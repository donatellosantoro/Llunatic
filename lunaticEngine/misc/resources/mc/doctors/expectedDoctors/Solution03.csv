#prescription
rxid,patient,npiref
P1,J. Doe,1112
P2,W. Smith,5555
P3,M. White,5556
_L119,J. Doe,1222

#doctor
npi,name,hospital,email
1112,R. Chase,Sacred Hearth,chase@sh.com
1222,C. Taub,_SKm1-N1(P1;J. Doe;1222;C. Taub),_SKm1-N2(P1;J. Doe;1222;C. Taub)
5555,G. House,PPTH,house@ppth.com
5556,_SKt0-N1(5556),_SKt0-N2(5556),_SKt0-N3(5556)

