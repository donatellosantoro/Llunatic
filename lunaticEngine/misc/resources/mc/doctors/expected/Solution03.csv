#prescription
rxid,patient,npiref
P3,M. White,5555
P1,J. Doe,_L537
P2,W. Smith,5555

#doctor
npi,name,hospital,email
_L138,_L215,Sacred Hearth,chase@sh.com
5555,G. House,PPTH,house@ppth.com
_L537,_SKt0-N1(1112),_SKt0-N2(1222),_SKt0-N3(1222)

